src/backings/structural/container.ts

```diff
@@ -1,10 +1,10 @@
-import {ObjectLike, Json} from "../../interface";
-import {ContainerType, CompositeType, Type, IJsonOptions} from "../../types";
-import {StructuralHandler} from "./abstract";
-import {toExpectedCase} from "../utils";
+import {Json, ObjectLike} from "../../interface";
+import {CompositeType, ContainerType, IJsonOptions} from "../../types";
 import {SszErrorPath} from "../../util/errorPath";
+import {toExpectedCase} from "../utils";
+import {StructuralHandler} from "./abstract";
 
-export class ContainerStructuralHandler<T extends ObjectLike> extends StructuralHandler<T> {
+export class ContainerStructuralHandler<T extends Record<string, unknown>> extends StructuralHandler<T> {
```

Comment:
ObjectLike

Reply:
that screws everything since it ObjectLike allows empty object without keys which casues keyof to stop working